#### 1.	Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16.  Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256


Найдём доверительный интервал используя формулу $$\overline{x} - Z_{\frac{a}{2}} \cdot \frac{\sigma}{\sqrt{n}} < a < \overline{x} + Z_{\frac{a}{2}} \cdot \frac{\sigma}{\sqrt{n}}$$

$\sigma = 16$  
$\overline{x} = M = 80$   
$n = 256$   
$Z_{\frac{a}{2}}$ определяется из таблицы и равно: $Z_{0.975}=1.96$

In [1]:
from math import sqrt

In [2]:
SP1 = 80 - 1.96*(16/sqrt(256))
SP2 = 80 + 1.96*(16/sqrt(256))

In [3]:
print(f'Доверительный интервал: {SP1} < a < {SP2}')

Доверительный интервал: 78.04 < a < 81.96


#### 2.	В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95


In [4]:
import numpy as np

In [5]:
x = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
n = len(x)
mu = x.mean()
p = 0.95
t = 2.262  # из таблицы Стьюдента
D = np.var(x,ddof=1) # Несмещённая оценка дисперсии
sigma = sqrt(D)

In [6]:
mu

6.590000000000001

In [7]:
SP1 = mu - (t*sigma/sqrt(n))
SP2 = mu + (t*sigma/sqrt(n))
print(f'Доверительный интервал: {SP1} < X < {SP2}')

Доверительный интервал: 6.267538255912426 < X < 6.912461744087575


In [8]:
print(f'Истинное значение величины X: {(SP1+SP2)/2:#.4}')

Истинное значение величины X: 6.590


#### 3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 мм

Примем гипотезу   H0: диаметр=17 мм, $\mu = \mu_0$   
обратная гипотеза H1: диаметр=17.5 мм $\mu > \mu_0$

In [9]:
n = 100
D = 4
sigma = 2
mu0 = 17
mu = 17.5
a = 0.05
ZT = 1.645 # из таблицы

In [10]:
ZR = (mu - mu0)*sqrt(n)/sigma
ZR

2.5

Поскольку $Z_{расч} > Z_{табл}$, нулевая гипотеза отвергается при уровне значимости 5%. Верна гипотеза H1

#### 4.Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [11]:
x = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
n = len(x)
n

10

In [12]:
mu = x.mean()
mu

198.5

Примем гипотезу   H0: вес пачки = 200 г, $\mu = \mu_0$   
обратная гипотеза H1: вес пачки = 198.5 г, $\mu < \mu_0$

In [13]:
n = 10
mu0 = 200
D = np.var(x,ddof=1) # Несмещённая оценка дисперсии
sigma = sqrt(D)
a = 0.01

По таблице критических точек распределения Стьюдента найдем критическую точку по
уровню значимости   
$\alpha = 0.01$ и числу степеней свободы $k = 9$ , откуда $T_{табл} \approx 3.25$

In [14]:
TR = (mu - mu0)*sqrt(n)/sigma
abs(TR)

1.0651074037450896

Поскольку $|T_{расч}| < T_{табл}$, следовательно нулевая гипотеза принимается.   
Утверждение продавца верно при доверительной вероятности 99%